### PR TITLE
remove references to presence lists

### PR DESCRIPTION
**Pull request by neilisfragile**
Relates https://github.com/matrix-org/matrix-doc/issues/1810

### CODE REVIEW

**Comment by turt2live** [comment]
Something like this surely needs an MSC as it's a spec change? It'd be even better if the feature were replaced by something more usable as well.

**Comment by neilisfragile** [comment]
[MSC](https://github.com/matrix-org/matrix-doc/pull/1818)

File: specification/modules/presence.rst [hunk 1]
```diff
@@ -26,14 +26,8 @@ Each user has the concept of presence information. This encodes:
...
 ``last_active``) and per-user (status) data, aggregated by the user's homeserver
-and transmitted as an ``m.presence`` event. This is one of the few events which
-are sent *outside the context of a room*. Presence events are sent to all users
-who subscribe to this user's presence through a presence list or by sharing
-membership of a room.
-
-A presence list is a list of user IDs whose presence the user wants to follow.
-To be added to this list, the user being added must be invited by the list owner
-who must accept the invitation.
+and transmitted as an ``m.presence`` event. Presence event are sent to
```
Comment (KitsuneRal) [inline specification/modules/presence.rst]: Either 'Presence event is' or 'Presence events are'.

**Comment by turt2live** [comment]
(landing this with anticipation that the FCPs are going to clear just fine - if this changes, we'll do the needed)

**Pull request merged**